packages/@lwc/integration-karma/scripts/karma-configs/test/sauce.js

```diff
@@ -51,26 +51,25 @@ const SAUCE_BROWSERS = [
 
     // Compat browsers
     {
-        label: 'sl_ie11',
-        browserName: 'internet explorer',
-        version: '11',
+        label: 'sl_edge_compat',
+        browserName: 'MicrosoftEdge',
+        version: '18',
```

Comment:
Do we assume that later versions of Edge will work if Chrome works? In other words, why is there no additional entry for Edge `latest-2`?

Reply:
Yeah I'm just assuming that if latest Edge will work, then latest Chrome will work. We could change that, but then we'd be running a lot more tests for non-compat mode (4 browsers instead of 3). I'm not sure it's really worth it.